package proto_ff;

import "yd_fieldoptions.proto";


message aiaihpDesc
{
	optional int64 skill = 1[(yd_fieldoptions.field_cname) = "技能"];
	optional int32 percent = 2[(yd_fieldoptions.field_cname) = "百分比"];
	optional int64 chat = 3[(yd_fieldoptions.field_cname) = "场景广播"];
}

message aiai
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "表ID"];
	optional int32 type = 2[(yd_fieldoptions.field_cname) = "ai类型"];
	optional string deadHateRemain = 3[(yd_fieldoptions.field_cname) = "死亡仇恨保留", (yd_fieldoptions.field_bufsize) = 32];
	optional string invisibleRemain = 4[(yd_fieldoptions.field_cname) = "隐身仇恨保留", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 isActive = 5[(yd_fieldoptions.field_cname) = "是否为主动怪"];
	optional int32 returnType = 6[(yd_fieldoptions.field_cname) = "回归类型"];
	optional int32 view = 7[(yd_fieldoptions.field_cname) = "视野"];
	optional int32 isAttackedShowOff = 8[(yd_fieldoptions.field_cname) = "是否关闭受击表现"];
	optional int32 pursueRange = 9[(yd_fieldoptions.field_cname) = "追击范围"];
	optional int32 actionRange = 10[(yd_fieldoptions.field_cname) = "怪物活动范围"];
	optional int32 wakeupRadius = 11[(yd_fieldoptions.field_cname) = "唤醒半径"];
	optional int32 isAIWorkWhenNobody = 12[(yd_fieldoptions.field_cname) = "是否无人时启用AI"];
	optional int32 bornEffectAIDelay = 13[(yd_fieldoptions.field_cname) = "出生AI延时"];
	optional string initSkills = 14[(yd_fieldoptions.field_cname) = "技能池初始技能组", (yd_fieldoptions.field_bufsize) = 74];
	repeated aiaihpDesc hp = 15[(yd_fieldoptions.field_cname) = "触发血量", (yd_fieldoptions.field_arysize) = 2];
}


message Sheet_aiai
{
	repeated aiai aiai_List = 1[(yd_fieldoptions.field_arysize)=166];
}
